services/dynamic-sidecar/tests/unit/test_models_shared_store.py

```diff
@@ -105,3 +109,18 @@ async def replace_list_in_shared_store(item: str):
         *(replace_list_in_shared_store(f"{x}") for x in range(PARALLEL_CHANGES))
     )
     assert len(shared_store.container_names) == PARALLEL_CHANGES
+
+
+async def test_data_format():
```

Comment:
we have a fixture in `pytest_simcore` called `project_tests_dir`. pls use that to get to mocks dir instead